webapp/javascript/components/Modals/ConfirmDelete/index.tsx

```diff
@@ -1,11 +1,43 @@
-import ShowModal from '@webapp/ui/Modals';
+import ShowModal, { ShowModalParams } from '@webapp/ui/Modals';
 
-function confirmDelete(object: string, onConfirm: () => void) {
+interface ConfirmDeleteProps {
+  objectType?: string;
+  objectName?: string;
+  object?: string;
+  warningMsg?: string;
+  onConfirm: () => void;
+  withConfirmationInput?: boolean;
+}
+
+function confirmDelete({
+  objectName,
+  objectType,
+  object,
```

Comment:
can we use  `objectName` prop instead of `object` ? since they have same `string` type

Reply:
fixed